### ISSUE TITLE
[Request] q-collapsible-item

### ISSUE THREAD

**Issue by slowaways** [quasarframework/quasar#1182]
I would like to have more control of how many label and sublabels a q-collapsible should have and how I would like to build it. Like q-item but for q-collapsible. See the example below:

Code:
```
    <q-list>
      <q-collapsible opened>
        <q-collapsible-item>
          <q-item-main>
            <q-item-tile sublabel">Foo</q-item-tile>
            <q-item-tile label>Bar</q-item-tile>
            <q-item-tile sublabel>3%</q-item-tile>
          </q-item-main>
          <q-item-side right>
            <q-item-tile label>$1200</q-item-tile>
            <q-item-tile sublabel color="positive">+0.22%</q-item-tile>
          </q-item-side>
        </q-collapsible-item>

        <q-item link to="/inbox/1">
          <q-item-side icon="mail" />
          <q-item-main label="Email 1" />
        </q-item>
      </q-collapsible>
    </q-list>
```
Result:

![spectacle j32547](https://user-images.githubusercontent.com/9668277/33231759-65fdd94c-d1d9-11e7-84b5-146556069ab4.png)

Currently you only have the option to add the one label and one sublabel through a property and no tile on the right:

Code:

`<q-collapsible label="Foo" sublabel="Bar">...</q-collapsible>`

Result:

![spectacle mtj852](https://user-images.githubusercontent.com/9668277/33231851-c016aafc-d1da-11e7-9efa-56f277089191.png)

**Comment by rstoenescu** [comment]
Hi, Will do for v0.15 through a scoped slot.

**Comment by rstoenescu** [comment]
Available in future v0.15 through "header" slot. Thanks for suggesting this.

**Comment by slowaways** [comment]
Awesome! Thanks!
